params/protocol_params.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+// <specular modification>
+// The L1 portion of the transaction fee accumulates at this predeploy
+var SpecularL1FeeRecipient = common.HexToAddress("0x2A0000000000000000000000000000000000001A")
```

Comment:
agree we should use the same approach for both `L1FeeRecipient` and `L2BaseFeeRecipient`, discussion here:

https://github.com/SpecularL2/go-ethereum/pull/13/files#r1406547980